test/docfx.Test/e2e/E2ETest.cs

```diff
@@ -125,6 +133,60 @@ private static async Task<(string docsetPath, E2ESpec spec)> CreateDocset(string
             return (docsetPath, spec);
         }
 
+        private static void AddExtendConfig(string docsetPath,  string[] suppressedRules)
+        {
+            var docfxConfigPath = Path.Combine(docsetPath, "docfx.yml");
+            if (File.Exists(docfxConfigPath))
+            {
+                string configContent = File.ReadAllText(docfxConfigPath);
+                if (string.IsNullOrWhiteSpace(configContent))
+                {
+                    GenerateSuppressedRulesFile(docfxConfigPath, suppressedRules);
+                    return;
+                }
+
+                var match = ExtendConfigRegex.Match(configContent);
+                if (match.Success)
+                {
+                    var orignalExtendValue = match.Groups[1].ToString().TrimStart(' ');
+                    var content = ExtendConfigRegex.Replace(configContent,
+                        $"extend:{Environment.NewLine}- {s_extendFileName}{(orignalExtendValue != string.Empty ? Environment.NewLine + "- " + orignalExtendValue : string.Empty)}");
+                    File.WriteAllText(docfxConfigPath, content);
```

Comment:
extend filed can be Expanded

Reply:
yes? and?

---
In reply to: [210170999](https://github.com/dotnet/docfx/pull/3196#discussion_r210170999) [](ancestors = 210170999)

Reply:
Expand means it can be array, or a single string value

---
In reply to: [210460350](https://github.com/dotnet/docfx/pull/3196#discussion_r210460350) [](ancestors = 210460350,210170999)